pkg/kv/dynamodb/main_test.go

```diff
@@ -1,23 +1,49 @@
 package dynamodb_test
 
 import (
+	"encoding/json"
 	"log"
 	"os"
 	"testing"
 
+	nanoid "github.com/matoous/go-nanoid/v2"
+	"github.com/treeverse/lakefs/pkg/kv/dynamodb"
 	"github.com/treeverse/lakefs/pkg/testutil"
 )
 
+const chars = "abcdef1234567890"
+
 var (
-	databaseURI string
+	dsn string
 )
 
+func testUniqueTableName() string {
+	return "kvstore_" + nanoid.MustGenerate(chars, 8)
+}
+
 func TestMain(m *testing.M) {
-	dbURI, cleanpFunc, err := testutil.RunLocalDynamoDBInstance("")
+	databaseURI, cleanpFunc, err := testutil.RunLocalDynamoDBInstance("")
 	if err != nil {
 		log.Fatalf("Could not connect to Docker: %s", err)
 	}
-	databaseURI = dbURI
+
+	testParams := &dynamodb.Params{
+		TableName:          testUniqueTableName(),
+		ReadCapacityUnits:  100,
+		WriteCapacityUnits: 100,
+		Endpoint:           databaseURI,
+		AwsRegion:          "us-east-1",
+		AwsAccessKeyID:     "fakeMyKeyId",
+		AwsSecretAccessKey: "fakeSecretAccessKey",
+	}
+
+	dsnBytes, err := json.Marshal(testParams)
```

Comment:
I'm pretty sure that's not what @nopcoder & @N-o-Z had in mind when they thought of the `dsn`. I believe that it would be super hard for the user to pass json configuration in a single string.

I understand why you did that though, there currently isn't an option to pass configuration to the KV implementation in any other way.

Reply:
The reason I went with that solution is, as you said, there is no `config` function, or configuration object to the store. It is pretty much the same for `postgres` where the dsn is actually the connection string (@N-o-Z please correctl me if I'm wrong), except for the `json` nature of it.
Anyhow, we can provide the parameters via env vars, via some `config` function (additional func in `Store` interface?) or leave it as it is, having the dsn and its options better documented in ortder to ease any future use of it.
We can also address it later, in a separate issue, as I believe there is going to be some discussion over it.
WDYT?

Reply:
Opened #3474